README.md

```diff
@@ -1 +1,59 @@
-# Mobile-menu
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet) 
+# Mobile-Menu
+
+On this milestone, we have created a menu for our prtfolio for the mobile version. To do this we used Javascript event listeners.
```

Comment:
- [ ] Kindly make sure to write Readme files without grammar errors. As you see, you misspelt `portfolio`. Please fix this grammar error in order to show, you are perfect in writing documentation.